### PR TITLE
Update readme.

### PR DESCRIPTION
**Pull request by homeofdonuts**
And some new files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# mytestrepo
\ No newline at end of file
+# mytestrepo
+This is my new repo.
```
Comment (test12aa) [inline README.md:2]: Ok cool.

File: c.md [hunk 2]
```diff
@@ -0,0 +1 @@
+New file 3.
```
Comment (test12aa) [inline c.md:1]: Great

File: b.md [hunk 3]
```diff
@@ -0,0 +1 @@
+New file 2.
```
Comment (test12aa) [inline b.md:1]: Fix this

Comment (homeofdonuts) [reply b.md]: Done.

**Pull request merged**